### ISSUE TITLE
byte-swapped client fails

### ISSUE THREAD

**Issue by idaho-1** [AsahiLinux/docs#96]
2023 Mac Mini M2 Pro Fedora 39
Firstly, congratulations and thanks for an amazing release!

When attempting to connect with PowerPC64 G5 based host or PowerPC G4 based host.
These are both big-endian, the connection is made but connection to X server fails:

lewis@fedora:~$ ssh -Y 10.1.200.13
(lewis@10.1.200.13) Password: 
lewis@pseudotsuga ~ $ xclock
Prohibited client endianess, see the Xserver man page Error: Can't open display: localhost:11.0

Message notwithstanding man page for Xserver and Xwayland do not reference byte-swapped client.
I note that this is disabled by default but can be activated with option "-byteswappedclients"
This appears to be already enabled:

lewis@fedora:~$ ps ax | grep wayland
   2129 tty2     Ssl+   0:00 /usr/libexec/gdm-wayland-session /usr/bin/gnome-session
   3372 ?        Sl     0:00 /usr/bin/Xwayland :0 -rootless -noreset -accessx -core -auth /run/user/1000/.mutter-Xwaylandauth.NM4SG2 -listenfd 4 -listenfd 5 -displayfd 6 -initfd 7 -byteswappedclients -enable-ei-portal
   6901 tty3     Ssl+   0:00 /usr/libexec/gdm-wayland-session /usr/bin/gnome-session
  13268 pts/5    S+     0:00 grep --color=auto wayland

Little-endian clients work as expected.

**Comment by DavidBuchanan314** [comment]
This is not an asahi linux issue

**Comment by idaho-1** [comment]
David,
Thanks for the link.
I checked the Fedora Wiki and installed 'gsettings' and 'dconf-editor' and turned on
the 'mutter' flag which turned on the 'Xwayland' option. I was a little confused
until I realized this is a "per-user" setting and I made the mistake of checking it as 'root'
where it was enabled but it failed for 'user' where it was not. I can confirm that this is not
a bug, it is a (slightly complicated) feature.
Byte swapped access to both a big-endian 64-bit G5 and big-endian 32-bit G4 _does_ work
from Asahi-Fedora-39. Sorry for the confusion.
